pyproject.toml

```diff
@@ -33,6 +33,16 @@ scikit-misc = "^0.1.4"
 plotnine = "^0.10.1"
 leidenalg = "^0.9.0"
 scipy = "^1.9.3"
+ete3 = "^3.1.2"
+pyqt5 = "^5.15.7"
+toytree = "^2.0.1"
+arviz = "^0.14.0"
+numpyro = "^0.10.1"
+adjusttext = "^0.7.3"
+statannotations = "^0.5.0"
+tasccoda = "^0.1.3"
```

Comment:
We should be able to remove tasccoda as a dependency, right? We reimplemented it!

Reply:
Yes, definitely! If not, we made a mistake^^